src/pages/__tests__/RegisterView.test.jsx

```diff
@@ -35,14 +37,23 @@ describe("Register View Component Tests", function() {
         expect(inputs).toBeInTheDocument();
     });
 
+    test("Invalid email typed, error message should be shown", () => {
+        const emailInput = queryByLabelText(renderedComponent.container, 'Email-input');
+
+        fireEvent.change(emailInput, { target: { value:  INVALID_EMAIL} });
+
+        const emailErrorMessage = queryByText(renderedComponent.container, ERROR_INVALID_EMAIL);
+        expect(emailErrorMessage).toBeInTheDocument();
+    });
+
     test("Valid inputs entered, fetch should be called", async () => {
         const emailInput = queryByLabelText(renderedComponent.container, 'Email-input');
         const passwordInput = queryByLabelText(renderedComponent.container, 'Password-input');
         const reenteredPasswordInput = queryByLabelText(renderedComponent.container, 'Reenter Password-input');
 
-        fireEvent.change(emailInput, { target: { value:  VALID_EMAIL} })
-        fireEvent.change(passwordInput, { target: { value: VALID_PASSWORD } })
-        fireEvent.change(reenteredPasswordInput, { target: { value: VALID_PASSWORD } })
+        fireEvent.change(emailInput, { target: { value:  VALID_EMAIL} });
```

Comment:
We really need to export our own fireChangeEvent. I'm tired of writing this target & value thing!

Reply:
So build our own class wrapper?

Reply:
Oh no! Maybe just a function that takes the input and the value. I'm all for convenience!